src/arch/io/network.hpp

```diff
@@ -9,6 +9,9 @@
 #include <arpa/inet.h>
 #include <netinet/in.h>
 
```

Comment:
We usually don't have extra newlines in the includes within the block of C include files. This newline should probably go away (the one after the openssl includes and before `<functional>` should stay though, since that's a new block that starts C++ includes)